Pr0gramm/Pr0gramm/Detail/DetailViewController.swift

```diff
@@ -134,6 +134,51 @@ class DetailViewController: ScrollingContentViewController, Storyboarded {
             setupVideo(for: item)
         }
     }
+    
+    func upvotePost() {
+        self.viewModel.vote(.up)
```

Comment:
self bitte nur wenn du es brauchst

Reply:
Ok, mein Codestil ist: Immer this/self verwenden, wenn Klassenvariablen verwendet werden, da man so leichter erkennen kann, welche Variablen im Methodenscope sind, und welche nicht.

Hat mich teilweise beim Lesen des Codes oft suchen lassen, wo was steht.

Reply:
Dafür hast du code highlighting. Für mich ist das boilerplate... :D

Reply:
Ja ok, ändere ich.

Ich komme aus der C#-Welt, da ist das branchenübliche Best Practice.

Reply:
Erledigt.

Reply:
![Bildschirmfoto 2021-05-24 um 22 08 22](https://user-images.githubusercontent.com/8708554/119403399-8f6d5200-bcde-11eb-9e68-1bcbba383673.png)

Das ist übrigens Syntaxhighlighting bei meinem Xcode. Unterschiede zwischen Klassen- und Methodenvariablen sehe ich nicht, das ist alles weiß.